store/orm/orm.go

```diff
@@ -49,6 +116,7 @@ func initializeDatabase(path string, duration time.Duration) (*storm.DB, error)
 }
 
 // GetBolt returns BoltDB from the ORM
+// TODO: We shouldn't be exposing this if we can avoid this
 func (orm *ORM) GetBolt() *bolt.DB {
```

Comment:
The underlying bolt instance is exposed for transactions. This particular method is used to remotely pull down a backup of the db: https://github.com/smartcontractkit/chainlink/blob/features/boot-wo-eth-connection/web/backup_controller.go#L15-L17

```
// Show streams a backup of the current db through a read-only transaction.
func (bc *BackupController) Show(c *gin.Context) {
	tx, err := bc.App.GetStore().GetBolt().Begin(false)
	if err != nil {
		c.AbortWithError(500, err)
		return
	}
	defer tx.Rollback()

	header := c.Writer.Header()
	header["Content-type"] = []string{"application/octet-stream"}
	header["Content-Disposition"] = []string{"attachment; filename=backup.bolt"}
	header["Content-Length"] = []string{strconv.Itoa(int(tx.Size()))}

	_, err = tx.WriteTo(c.Writer)
	if err != nil {
		c.AbortWithError(500, err)
	}
}
```

Do we want the postgres implementation to also support this? That might be a good thing to add to the spike.